### PR TITLE
[Issue_8] Add 'FSF All Permissions License'

### LINKED ISSUE #8: Add "FSF All Permissions License"
This is the only one used in WildFly / WildFly Core that isn't in KnownLicenses

### PR DESCRIPTION
**Pull request by bstansberry**
Resolves #8

### CODE REVIEW

**Comment by bstansberry** [comment]
@michpetrov Please review

**Review by jamezp — APPROVED** [review]

File: src/main/java/org/wildfly/maven/plugins/licenses/KnownLicenses.java [hunk 1]
```diff
@@ -144,6 +148,8 @@ public class KnownLicenses {
         KNOWN_LICENSES.put("epl 2.0", EPL_2_0_LICENSE);
         KNOWN_LICENSES.put("epl-2.0", EPL_2_0_LICENSE);
 
+        KNOWN_LICENSES.put(FSF_ALL_NAME.toLowerCase(), FSF_ALL_LICENSE);
```
Comment (jamezp) [inline src/main/java/org/wildfly/maven/plugins/licenses/KnownLicenses.java:151]: The others aren't like this, but might be worth considering adding `toLowerCase(Locale.ROOT)`.

**Review by michpetrov — APPROVED** [review]
I'd go with `https` to keep it in line with others. Otherwise LGTM.

File: src/main/java/org/wildfly/maven/plugins/licenses/KnownLicenses.java (hunk 1 above)
Comment (michpetrov) [reply src/main/java/org/wildfly/maven/plugins/licenses/KnownLicenses.java:151]: @jamezp does it matter? I don't expect the license names to contain non-ASCII characters.

Comment (jamezp) [reply src/main/java/org/wildfly/maven/plugins/licenses/KnownLicenses.java:151]: Probably not. I would assume it wouldn't either TBH, but it's just always safe :)

Comment (bstansberry) [reply src/main/java/org/wildfly/maven/plugins/licenses/KnownLicenses.java:151]: @jamezp I filed https://github.com/wildfly/licenses-plugin/issues/12 as a 'good-first-issue'.

**Comment by bstansberry** [comment]
> I'd go with `https` to keep it in line with others. Otherwise LGTM.

Thanks for catching that. I fixed it and pushed.

**Comment by jamezp** [comment]
LGTM

**Comment by bstansberry** [comment]
Thanks @michpetrov and @jamezp

**Pull request merged**